blog-content/articles/testing-makes-you-faster-day-one.md

```diff
@@ -7,13 +7,10 @@ summary: How testing makes you fast day one and not in some unknown future
 
 # Tests Are Here To Speed You Up
 
-*"I am not writing tests right now as they are going to slow me down"* is one of the most common excuses I heard on 
-the projects I worked. 
+*"I am not writing tests right now as they are going to slow me down"* is one of the most common excuses I heard in 
+the projects I worked on.
 
-One possible reason for being slowed down by tests is the lack of tests writing experience. 
-
-But I think there is another, bigger reason or rather untapped, most likely unrealized super power. The power of exercising
-any piece of your software in `O(1)` keystrokes.
+A lack of test writing experience can make you feel this way, but I think there is another, bigger reason for the perception that tests slow you down. I believe this reason is actually an untapped superpower: the power of exercising any piece of your software in `O(1)` keystrokes. 
```

Comment:
My sentence here is a suggestion, not a mandate :D But also, I recommend you rephrase your sentence. I think the key ideas are:
1. Not having experience writing tests is often the reason people give to explain why they feel like tests slow them down, but it is not the root cause - not knowing that you can build teleports is a bigger reason people feel like tests slow them down.
2. Knowing how to write teleports is an untapped superpower
3. Writing teleports lets you exercise any piece of software in `O(1)` keystrokes

Sidenote: where does O(1) keystrokes come from?

Reply:
This makes sense. I need help expressing my ideas. You got the main points right. 
By o(1) I meant to say a constant time that doesn’t send in project size but also that you can literally press one key to run a test

Reply:
That's great! I strongly recommend you work the above explanation into the text, so that it's clear to the reader where the O(1) comes from.